internal/dinosaur/pkg/api/dbapi/central_request_types.go

```diff
@@ -51,6 +52,9 @@ type CentralRequest struct {
 	RoutesCreationID string `json:"routes_creation_id"`
 	// DeletionTimestamp stores the timestamp of the DELETE api call for the resource
 	DeletionTimestamp *time.Time `json:"deletionTimestamp"`
+
+	// All we need to integrate Central with an IdP.
+	AuthConfig
```

Comment:
```suggestion
	AuthConfig AuthConfig `json:"authConfig"`
```
You don't seem to want to use this as an embedded struct field anyway

Reply:
I'm not sure about this. Look how any non-basic type is represented as `api.JSON`, e.g., `Routes`. I definitely don't want to store `AuthConfig` as JSON here nor I fancy figuring out why we don't simply use nested structs.

So yeah, I want to embed this. Would you suggest dropping `AuthConfig` when accessing its fields?